app/client/cli/keys.go

```diff
@@ -0,0 +1,465 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/pokt-network/pocket/app/client/keybase"
+	"github.com/spf13/cobra"
+)
+
+var (
+	outputFile string
+	inputFile  string
+	exportAs   string
+	importAs   string
+	hint       string
+	newPwd     string
+)
+
+func init() {
+	rootCmd.AddCommand(NewKeysCommand())
+}
+
+func NewKeysCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Keys",
+		Short:   "Key specific commands",
+		Aliases: []string{"keys"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	createCmds := keysCreateCommands()
+	deleteCmds := keysDeleteCommands()
+	updateCmds := keysUpdateCommands()
+	getCmds := keysGetCommands()
+	exportCmds := keysExportCommands()
+	importCmds := keysImportCommands()
+	signCmds := keysSignCommands()
+
+	// Add --pwd and --hint flags
+	applySubcommandOptions(createCmds, attachPwdFlagToSubcommands())
+	applySubcommandOptions(createCmds, attachHintFlagToSubcommands())
+
+	// Add --pwd flag
+	applySubcommandOptions(deleteCmds, attachPwdFlagToSubcommands())
+
+	// Add --pwd and --hint flags
+	applySubcommandOptions(updateCmds, attachPwdFlagToSubcommands())
+	applySubcommandOptions(updateCmds, attachNewPwdFlagToSubcommands())
+	applySubcommandOptions(updateCmds, attachHintFlagToSubcommands())
+
+	// Add --pwd, --output_file and --export_format flags
+	applySubcommandOptions(exportCmds, attachPwdFlagToSubcommands())
+	applySubcommandOptions(exportCmds, attachOutputFlagToSubcommands())
+	applySubcommandOptions(exportCmds, attachExportFlagToSubcommands())
+
+	// Add --pwd, --hint, --input_file and --import_format flags
+	applySubcommandOptions(importCmds, attachPwdFlagToSubcommands())
+	applySubcommandOptions(importCmds, attachHintFlagToSubcommands())
+	applySubcommandOptions(importCmds, attachInputFlagToSubcommands())
+	applySubcommandOptions(importCmds, attachImportFlagToSubcommands())
+
+	// Add --pwd flag
+	applySubcommandOptions(signCmds, attachPwdFlagToSubcommands())
+
+	cmd.AddCommand(createCmds...)
+	cmd.AddCommand(deleteCmds...)
+	cmd.AddCommand(getCmds...)
+	cmd.AddCommand(exportCmds...)
+	cmd.AddCommand(importCmds...)
+	cmd.AddCommand(signCmds...)
+
+	return cmd
+}
+
+func keysCreateCommands() []*cobra.Command {
+	cmds := []*cobra.Command{
+		{
+			Use:     "Create",
+			Short:   "Create new key",
+			Long:    "Creates a new key and stores it in the keybase",
+			Aliases: []string{"create"},
+			Args:    cobra.ExactArgs(0),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				// Open the debug keybase at the specified path
+				pocketDir := strings.TrimSuffix(dataDir, "/")
+				keybasePath, err := filepath.Abs(pocketDir + keybaseSuffix)
+				if err != nil {
+					return err
+				}
+				kb, err := keybase.NewKeybase(keybasePath)
+				if err != nil {
+					return err
+				}
+
+				pwd = readPassphrase(pwd)
+
+				kp, err := kb.Create(pwd, hint)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("Key created: %s\n", kp.GetAddressString())
```

Comment:
```suggestion
				logger.Global().Str("address", kp.GetAddressString()).Msg("New Key Created")
```

Leaving one comment but ditto everywhere else